src/games/strategy/net/ServerMessenger.java

```diff
@@ -717,24 +717,24 @@ public boolean isServer() {
 
   @Override
   public void removeConnection(final INode node) {
-    if (node.equals(m_node)) {
+    if (node.equals(this.node)) {
```

Comment:
Can we rename this local variable to avoid having both local and member variables with the same name?